Source/EasyNetQ/Producer/IResponderConfiguration.cs

```diff
@@ -10,12 +11,20 @@ public class ResponderConfiguration : IResponderConfiguration
     {
         public ResponderConfiguration(ushort defaultPrefetchCount)
         {
+            Priority = 0;
             PrefetchCount = defaultPrefetchCount;
         }
 
+        public int Priority { get; private set; }
```

Comment:
@jmalStorm  I suggest using `byte?` here to provide additional information about recommended maximum priority size (0-255). Also `IPublishConfiguration`  in  `bus.Publish(...)`  already has byte priority so no reason to make it int.